com.unity.multiplayer.transport.enet/Runtime/EnetTransport.cs

```diff
@@ -362,24 +361,23 @@ public override void Init()
                 internalChannels.Add(i, new EnetChannel()
                 {
                     Id = i,
-                    Name = MLAPI_CHANNELS[i].Name,
                     Flags = MLAPIChannelTypeToPacketFlag(MLAPI_CHANNELS[i].Type)
                 });
             }
 
             // Internal Channels
             for (int i = 0; i < Channels.Count; i++)
             {
-                byte id = (byte)(i + MLAPI_CHANNELS.Length);
-
-                channelIdToName.Add(id, Channels[i].Id);
-                channelNameToId.Add(Channels[i].Id, id);
-                internalChannels.Add(id, new EnetChannel()
-                {
-                    Id = id,
-                    Name = Channels[i].Name,
-                    Flags = Channels[i].Flags
-                });
+                byte id = (byte)(i + MLAPI_CHANNELS.Length); // ugh
```

Comment:
@TwoTenPvP I could use a little guidance on this one...it looks like eNet has a second set of Internal channels separate from the ones defined in MLAPI?  And before they were 'scoped' by just offsetting the # of channels in MLAPI.  I'm trying to get where these internal channels are coming from.  Or maybe they're not used?